interface/patient_file/encounter/forms.php

```diff
@@ -632,10 +638,22 @@ function divtoggle(spanid, divid) {
         echo "<div class='form_header_controls'>";
 
         // If the form is locked, it is no longer editable
-        if ( $esign->isLocked() ) {
-            echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-".attr($formdir)."-".attr($iter['id'])."'><span>".xlt('Locked')."</span></a>";
+        if ($esign->isLocked()) {
+             $flag = false;
+             if($GLOBALS['enable_group_therapy']){
+                 if($acl_groups) $flag = true;
+             }
+             else{
+                 $flag = true;
+             }
+
+             if($flag){
+                 echo "<a href=# class='css_button_small form-edit-button-locked' id='form-edit-button-" . attr($formdir) . "-" . attr($iter['id']) . "'><span>" . xlt('Locked') . "</span></a>";
+             }
```

Comment:
OK but suppose (for example) this is the Vitals form and group therapy is enabled and the user does not have glog permission. It appears they will not see that the Vitals form is locked. Similarly for the code below, editing or e-signing the Vitals form will depend on glog permission. Why?